packages/fast-tooling/package.json

```diff
@@ -0,0 +1,86 @@
+{
+    "name": "@microsoft/fast-tooling",
+    "description": "FAST Tooling",
+    "sideEffects": false,
+    "version": "0.0.0",
+    "author": {
+      "name": "Microsoft",
+      "url": "https://discord.gg/FcSNfg4"
+    },
+    "license": "MIT",
+    "repository": {
+      "type": "git",
+      "url": "git+https://github.com/Microsoft/fast-dna.git"
+    },
+    "bugs": {
+      "url": "https://github.com/Microsoft/fast-dna/issues/new/choose"
+    },
+    "main": "dist/index.js",
+    "types": "dist/index.d.ts",
+    "scripts": {
+      "build": "tsc -p ./tsconfig.build.json",
+      "clean:dist": "node ../../build/clean.js dist",
+      "prepare": "yarn clean:dist && yarn build",
+      "prettier": "prettier --config ../../.prettierrc --write \"**/*.ts\"",
+      "prettier:diff": "prettier --config ../../.prettierrc \"**/*.ts\" --list-different",
+      "test": "yarn tslint && yarn unit-tests",
+      "tslint": "tslint -c ./tslint.json '**/*.ts'",
+      "tslint:fix": "tslint -c ./tslint.json --fix '**/*.ts'",
+      "unit-tests": "jest --runInBand",
+      "watch": "yarn build -- -w --preserveWatchOutput"
+    },
+    "jest": {
+      "collectCoverage": true,
+      "coverageThreshold": {
+        "global": {
+          "statements": 80,
+          "branches": 65,
+          "functions": 80,
+          "lines": 80
+        }
+      },
+      "coveragePathIgnorePatterns": [
+        "/(.tmp|__tests__)/*"
+      ],
+      "testURL": "http://localhost",
+      "transform": {
+        "^.+\\.ts$": "ts-jest",
+        "^.+\\.js$": "babel-jest"
+      },
+      "transformIgnorePatterns": [
+        "!<rootDir>/node_modules/lodash-es"
+      ],
+      "testRegex": "(\\.|/)(test|spec)\\.(js|ts)$",
+      "moduleFileExtensions": [
+        "ts",
+        "js",
+        "json"
+      ]
+    },
+    "devDependencies": {
+      "@microsoft/fast-tslint-rules": "^3.0.4",
```

Comment:
Can you audit these package? Several of them don't seem to be necessary, like lodash-es and enzyme